.claude/hooks/turn-reset.sh

```
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -uo pipefail
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"; . "$DIR/_lib.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./_lib.sh was not specified as input (see shellcheck -x).

(SC1091)